src/olympia/migrations/878-add-feature-compatibility-table.sql

```diff
@@ -6,3 +6,4 @@ CREATE TABLE `addons_addonfeaturecompatibility` (
 )
 ;
 ALTER TABLE `addons_addonfeaturecompatibility` ADD CONSTRAINT `addon_id_refs_id_7779cd14` FOREIGN KEY (`addon_id`) REFERENCES `addons` (`id`);
+
```

Comment:
Please keep this old migration intact, no need to modify it that's confusing.

Reply:
I have copied the old migration and committed.